lib/traject/config/sirsi_config.rb

```diff
@@ -1803,6 +1803,8 @@ def holdings(record, context)
         call_number: (item['a'] || '').strip,
         current_location: item['k'],
         home_location: item['l'],
+        location: 'PLACEHOLDER',
+        status: 'Available',
```

Comment:
```suggestion
```

I'm not sure if you've added these because they're now required, but we shouldn't populate them with wrong data either.

Reply:
what would be appropriate values?

Reply:
`nil`?